### DISCUSSION TITLE
Disable Groups by Region

### DISCUSSION THREAD

**Discussion by HofferDoffer** [code4recovery/12-step-meeting-list#1112] [Ideas]
It would be a nice feature to disable/deactivate a list of meetings by Region. We have a need to do that on our website - need to disable until we get more accurate (post-Covid) information from one Region. 
Currently, the only option is to delete the Region.

**Comment by joshreisner** [comment]
i _think_ you should be able to do that? we typically deactivate meeting by setting them to draft. i believe this should be possible by filtering the admin view of meetings by region, then selecting all, then edit, then setting them to status > draft

<img width="1743" alt="Screenshot 2023-04-05 at 10 56 44 AM" src="https://user-images.githubusercontent.com/1551689/230164334-4133d968-e04c-45f3-88b8-b50cb1afcebb.png">
